apps/website/src/pages/docs/footer.mdx

```diff
@@ -14,14 +14,54 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-footer' />
-
 <LiveExample src='Footer.main.tsx' truncate={false}>
   <AllExamples.FooterMainExample client:load />
 </LiveExample>
 
 The footer is fixed to the bottom of a web page, and contains your typical legally required links. The legal footer is only required to be displayed on pages that are considered a “main entry point into your application”. This means that the sign in and home page of any hub / app are good places to display the footer. You do not need to display the footer within an App UI widget, or deeper pages within an app. You can not include the footer on a page with infinite scrolling because you can not reach the bottom of the page.
 
+## Variants
```

Comment:
This section should be renamed and probably have some text explanation. Currently it feels too robotic - the examples that follow are not "variants", they are just different customizations to the default footer.

Reply:
I am kind of confused about what needs to be changed here. Here is my understanding: 
1) Change the name of the header "Variants" to something suitable?
2) Do you want me to add some info underneath "Variants" or change this text: The footer is fixed...  ...bottom of the page?

Reply:
I think you might be confused by github's UI. When looking at review comments in the Conversation tab, only the last line is relevant. Github shows a few lines above it, but those are unrelated to the comment.

<img width="743" alt="" src="https://github.com/iTwin/iTwinUI/assets/9084735/671411d6-72fb-44b6-8f61-e4085021dd12">

So yes, change the heading and add some text under the heading.

Reply:
Makes sense. Thanks

Reply:
generally it's best to keep headings short, so try just "Customizations"

Reply:
Fixed